wagtail_review/admin_api/serializers.py

```diff
@@ -1,16 +1,36 @@
+from django.contrib.auth import get_user_model
 from rest_framework import serializers
 
 from .. import models
 from ..api.serializers import CommentSerializer
 
 
+class UserSerializer(serializers.ModelSerializer):
+    id = serializers.ReadOnlyField(source='pk')
+    full_name = serializers.ReadOnlyField(source='get_full_name')
```

Comment:
Just noting that this is optional for a custom user model, as per https://docs.djangoproject.com/en/3.0/topics/auth/customizing/. Is this relied upon?

Reply:
Fixed in latest commit